### ISSUE TITLE
[FR] - Allow Auto Unpack of Multi Rar torrents.

### ISSUE THREAD

**Issue by veenarm** [kubealex/k8s-mediaserver-operator#25]
**Is your feature request related to a problem? Please describe.**
unrar capabilities of Transmission to allow sonarr/radarr to auto move torrents that are multi packaged rar files.

**Describe the solution you'd like**
Using the rtorrent config file update and adding a configMap shell script with correct permissions, you can have a small shell script run to unrar the files once downloading has completed.

**Describe alternatives you've considered**
none

**Additional context**
I've got it working locally with direct modifications to the ouput yamls, changes are needed to the Operator to allow the config updates and Volumes/Mounts. I am happy to do a PR for this work.

**Comment by kubealex** [comment]
Hi @veenarm, is this some kind of use case that can happen while fetching stuff from sonarr/radarr?

**Comment by veenarm** [comment]
Wow, I totally forgot about this (even though I've been using it in my homelab now since this post).

I forked your Operator and made the following changes: https://github.com/veenarm/k8s-mediaserver-operator/commit/eb20cce288c285c0bacf5d2ee1f32cf07f78d103  
- Ignore the typo on line 124 in `values.yaml` should be `unrar.sh` not `unpack.sh`

The use case is that sometimes you may download movies/tv shows with sonarr/radarr that aren't the actual file (they're compress zipped in multiple files). Sonarr/Radarr can't copy the files to it's expected location after download because of this, so in my commit above I have a script that auto unpacks after download completion so they can be transferred to there new home.

The other use-case is the next part is to execute a script `after finished seeding` so that it removes the torrent and deletes it's data from the transmission downloads.

**Comment by veenarm** [comment]
I've since started using : https://github.com/Unpackerr/unpackerr
It intergrates with sonarr/radarr nicely.

**Comment by Nicxx2** [comment]
In case you find this useful, you can use this: https://github.com/Nicxx2/auto-unrar
  
It is an easy setup if you are using docker, or you can just adjust and use the script.
  
Basically, it will automatically extract RAR files, and there are various options you can include such as: extracting files to a specific directory, deleting rar file after extraction etc.

**Comment by veenarm** [comment]
Yes, however Unpackerr can be integrated with webhooks from your Prowlarr/Sonarr/Radarr installers etc.. so it's waiting for the new data to actually arrive, and then when it's completed it I believe tells them as such :)
